runsc/cmd/exec.go

```diff
@@ -140,6 +144,35 @@ func (ex *Exec) Execute(_ context.Context, f *flag.FlagSet, args ...any) subcomm
 		log.Infof("Using exec capabilities from container: %+v", e.Capabilities)
 	}
 
+	// Create the file descriptor map for the process in the container.
+	fdMap := map[int]*os.File{
+		0: os.Stdin,
+		1: os.Stdout,
+		2: os.Stderr,
+	}
+
+	// Add custom file descriptors to the map.
+	var files []*os.File
+	for _, fd := range ex.passFDs {
+		file := os.NewFile(uintptr(fd), "")
+		if file == nil {
+			util.Fatalf("failed to create file from file descriptor %d", fd)
+		}
+		fdMap[int(file.Fd())] = file
+	}
+
+	// Close the underlying file descriptors after we have passed them.
+	defer func() {
+		for _, file := range files {
```

Comment:
We never add any files to the `files` slice.  We should be iterating over the `fdMap` here, right?

```
for _, file := range fdMap {
 ...
}
```

We do this correctly in `run.go`.

Reply:
Oh, yes. Of course. I have changed this.

https://github.com/google/gvisor/blob/c66f9f08bdae58f94ff2540a9d7b2acaa799bb9b/runsc/cmd/exec.go#L163-L171